database/db.py

```diff
@@ -73,9 +73,9 @@ def createTables(self):
 
         #Creates queries for each table
         createCustomers = "CREATE TABLE Customers (customer_id int NOT NULL, f_name varchar(255) NOT NULL,l_name varchar(255) NOT NULL, email varchar(255) NOT NULL, password varchar(255) NOT NULL, PRIMARY KEY(customer_id), UNIQUE KEY(email));"
-        createProducts = "CREATE TABLE Products (product_id int, name varchar(255), PRIMARY KEY(product_id), UNIQUE KEY(name));"
+        createProducts = "CREATE TABLE Products (product_id int, name varchar(255), msrp varchar(32), PRIMARY KEY(product_id), UNIQUE KEY(name));"
         createRetailers = "CREATE TABLE Retailers (retailer_id int, name varchar(255), PRIMARY KEY(retailer_id), UNIQUE KEY(name));"
-        createRetailers_Products = "CREATE TABLE Retailers_Products (product_id int, retailer_id int, name varchar(64), price varchar(32), photo varchar(255), source varchar(255), instock varchar(32), new varchar(32), price_check varchar(64), PRIMARY KEY (product_id, retailer_id), FOREIGN KEY (product_id) REFERENCES Products(product_id), FOREIGN KEY (retailer_id) REFERENCES Retailers(retailer_id))"
+        createRetailers_Products = "CREATE TABLE Retailers_Products (product_id int, retailer_id int, name varchar(64), price varchar(32), msrp varchar(32), photo varchar(255), source varchar(255), instock varchar(32), new varchar(32), price_check varchar(64), PRIMARY KEY (product_id, retailer_id), FOREIGN KEY (product_id) REFERENCES Products(product_id), FOREIGN KEY (retailer_id) REFERENCES Retailers(retailer_id))"
```

Comment:
Thank you for adding this in! I forgot to do it in my PR and need to make a follow up to add product_type in :(

Reply:
I went ahead and added product_type into the db because it was breaking the product_type = x[10] I had above